kolibri/plugins/coach/assets/src/views/plan/assignments/AssignmentDetailsModal.vue

```diff
@@ -257,6 +257,7 @@
     },
     watch: {
       title() {
+        this.title = this.title.trim();
```

Comment:
Sorry, missed this on first read - this would trigger two emits (although both of the same value) - would be better to just use `v-model.trim` on the form input binding in the component: https://v2.vuejs.org/v2/guide/forms#trim